prototype.tf

```diff
@@ -0,0 +1,147 @@
+# Specify the provider, here
+provider "google" {
+  project                 = var.project_id
+  region                  = var.compute_region
+}
+
+/* 
+ * [BEGIN] GCS Resources
+ */
+
+# Raw data landing zone for data ingestion
+resource "google_storage_bucket" "gcs_data_ingestion_landing_bucket" {
+  name                    = var.gcs_landing_bucket
+  location                = var.gcs_region
+  force_destroy           = true # This forces deletion of objects created in bucket post provisioning
```

Comment:
When you create a bucket in Terraform, by default `force_destroy` is set to false. What this means is that if there are objects created and placed in the bucket after TF sets up the bucket, when trying to do a terraform destroy it will error and refuse to delete the bucket. It's a safeguard. Setting this to true allows you to destroy the bucket with objects in it. I set it here since it's a sandbox/dev environment and I frequently destroy and recreate the environment.

https://www.terraform.io/docs/providers/google/r/storage_bucket.html#force_destroy